Makefile

```diff
@@ -15,7 +15,7 @@ deps: core/zmiddleware.go core/dnsserver/zdirectives.go
 	go get -u github.com/golang/lint/golint
 
 .PHONY: check
-check: fmt deps
+check: deps
```

Comment:
OK - mystery solved... at some point I accidentally dropped the "docker" target off a  `make -f Makefile.release docker` command ...
`make -f Makefile.release` automatically does a `git commits -am"Release $(VERSION)"` ...